bouture.js

```diff
@@ -76,7 +88,7 @@ function completeElement (tag, args) {
                 const options = {once: keyName.match(/^once/)}
                 element.addEventListener(eventName, eventValue, options)
               }
-            } else {
+            } else if (isAttribute(keyName)) {
```

Comment:
I'm uncomfortable with `else if`, but if the linter and you are fine with it, i'm okay

Reply:
Maybe it would be worth adding a `console.warn` to say the `tag`+`attribute` pair is not considered valid by Bouture and is ignored